privaterelay/templates/profile.html

```diff
@@ -18,6 +18,13 @@
   data-fxa-subscriptions-url="{{ settings.FXA_SUBSCRIPTIONS_URL }}" 
   data-premium-prod-id="{{ settings.PREMIUM_PROD_ID }}" 
   data-premium-price-id="{{ settings.PREMIUM_PRICE_ID }}" 
+  {% if settings.PREMIUM_ENABLED %}
+    {% with request.user.profile_set.first as user_profile %}
+      {% if user_profile.subdomain %}
+        data-premium-subdomain-set="true"
```

Comment:
Rather than just setting `true`, can I get an output from the variable? Similar to `var foo = (x > y)` in JS?

Reply:
Yup, the `user_profile.subdomain` value should be a string of the user's subdomain. So it could be:
```suggestion
        data-premium-subdomain-set="{{ user_profile.subdomain }}"
```